src/hotspot/share/code/codeCache.cpp

```diff
@@ -1559,6 +1559,33 @@ void CodeCache::log_state(outputStream* st) {
             unallocated_capacity());
 }
 
+void CodeCache::write_perf_map(outputStream* st) {
+  MutexLocker mu(CodeCache_lock, Mutex::_no_safepoint_check_flag);
+
+  char fname[32];
+  jio_snprintf(fname, sizeof(fname), "/tmp/perf-%d.map", os::current_process_id());
+
+  fileStream fs(fname, "w");
+  if (!fs.is_open()) {
+    st->print_cr("Failed to create %s", fname);
```

Comment:
This message should be used Unified Logging.

Reply:
`st->print_cr()` is still needed? and also I've intended to replace all `print_cr()` to UL (L1582 and L1587)